charts/gha-runner-scale-set/values.yaml

```diff
@@ -163,9 +163,19 @@ template:
   ##                   storage: 1Gi
   spec:
     containers:
-    - name: runner
-      image: ghcr.io/actions/actions-runner:latest
-      command: ["/home/runner/run.sh"]
+      - name: runner
+        image: ghcr.io/actions/actions-runner:latest
+        command: ["/home/runner/run.sh"]
+
+## template is the PodSpec for each listener Pod
+# listenerTemplate:
+#   spec:
+#     containers:
+#     - name: listener # merge extra container configuration to the default configuration for the listener container.
```

Comment:
What happens when end-users change this value?

Reply:
It will be a side-container then